apps/backend/models/parent.ts

```diff
@@ -0,0 +1,65 @@
+import { User } from './user';
+
+import { list } from '@keystone-6/core';
+import { text, integer, relationship, json, timestamp } from '@keystone-6/core/fields';
+
+export const Parent = list({
+  //only logged in users can create, update, and delete
+  access: {
+    operation: {
+      query: () => true, //anyone can query parent data
+      create: ({ session }) => !!session,
+      update: ({ session }) => !!session,
+      delete: ({ session }) => !!session,
+    },
+    filter: {
+      query: ({ session }) =>
+        session?.data.isAdmin ? true : { user: { id: { equals: session?.data.id } } }, //allows users to see their own parents
+    },
+    item: {
+      //Allows updates/deletes if user is admin or own parent
+      update: ({ session, item }) => session?.data.isAdmin || item.userId === session?.data.id,
+      delete: ({ session, item }) => session?.data.isAdmin || item.userId === session?.data.id,
```

Comment:
Security consideration here: the access control uses `item.userId` but the actual field name in the database is `user` (nullable string)

```suggestion
      update: ({ session, item }) => session?.data.isAdmin || item.user === session?.data.id,
      delete: ({ session, item }) => session?.data.isAdmin || item.user === session?.data.id,
```